docs/versioned_docs/version-zenith/quickstart/842901-test.mdx

```diff
@@ -0,0 +1,56 @@
+---
+slug: /quickstart/842901/test
+hide_table_of_contents: true
+title: "Add a function to test the application"
+---
+
+import PartialExperimentalDocs from '../partials/_experimental.mdx';
+
+# Quickstart
+
+<PartialExperimentalDocs />
+
+## Add a function to test the application
+
+The function in the previous example was illustrative - you wouldn't ever use it in the real world! So, the next step is to write a Dagger function to do something more useful, like running an application's tests.
+
+Add a new `Test()` function to `main.go`, which runs the application's unit test suite:
+
+This function does the following:
+
+- It uses the `dag` client's `Container().From()` method to initialize a new container from a base image. In this example, the base image is again `node:18-slim`. This method returns a `Container` initialized from that base image.
+- When a module is loaded into Dagger, it includes the directory tree at which `dagger.json` is located. In this example, since `dagger.json` is co-located with the application source code directory, that directory is auto-included in the module container's filesystem and can be accessed via `dag.Host().Directory()`.
```

Comment:
```suggestion
- When a module is loaded into Dagger, it includes the directory tree at which `dagger.json` is located. In this example, since `dagger.json` is co-located with the application source code directory, that directory is auto-included in the module container's filesystem and can be accessed via `dag.CurrentModule().Source()`.
```

Reply:
This example was moved to https://github.com/dagger/dagger/pull/6504 so I've made the change there.